region/crateria/central/Parlor and Alcatraz.json

```diff
@@ -1949,6 +2030,104 @@
         "Note that the global Geemer may still be in this area if traversing the room quickly, and can hit Samus off-screen."
       ]
     },
+    {
+      "link": [5, 8],
+      "name": "Direct G-Mode Morph Spark Interrupt (Crystal Flash)",
+      "entranceCondition": {
+        "comeInWithGMode": {
+          "mode": "direct",
+          "morphed": true
+        }
+      },
+      "requires": [
+        "h_ZebesIsAwake",
+        {"not": "f_ZebesSetAblaze"},
+        "h_artificialMorphMovement",
+        {"or": [
+          "canTrickyDodgeEnemies",
+          {"enemyDamage": {"enemy": "Geemer (blue)", "type": "contact", "hits": 1}}
+        ]},
+        "h_artificialMorphCrystalFlash",
+        {"canShineCharge": {
+          "usedTiles": 25,
+          "steepUpTiles": 3,
+          "steepDownTiles": 3,
+          "openEnd": 0
+        }},
+        "canBeVeryPatient",
```

Comment:
```suggestion
```

- with morph -> farm and then go into the alcatraz tunnel to fix the camera and use the right geemer
- takes 2 min 40s to get the geemers into position, so theres no concern of ibj being too slow to get into position. I think this probably can be lowered to canBePatient (can be sped up with a super, but not to below 90s)
- you can also use the ripper if you need to, but that isnt trivial to get into position
- I wonder if the top left geemer might be a better one to keep alive. I had no problem getting over 99E, probably because the first 50 are while in health-bomb, and you can do some farming while close to guarentee drops past that as well.

TLDR
Seems fine to just remove this and edit the description a bit